client_integration_test.go

```diff
@@ -0,0 +1,184 @@
+package notify_test
+
+import (
+	"os"
+	"testing"
+
+	notify "github.com/govau/notify-client-go"
+)
+
+func setup(t *testing.T) (*notify.Client, string, string) {
+	apiKey := os.Getenv("API_KEY")
+	if apiKey == "" {
+		t.Fatal("API_KEY environment variable not set")
+	}
+
+	smsTemplateId := os.Getenv("SMS_TEMPLATE_ID")
+	if smsTemplateId == "" {
+		t.Fatal("SMS_TEMPLATE_ID environment variable not set")
+	}
+
+	emailTemplateId := os.Getenv("EMAIL_TEMPLATE_ID")
+	if emailTemplateId == "" {
+		t.Fatal("EMAIL_TEMPLATE_ID environment variable not set")
+	}
+
+	var err error
+	client, err := notify.NewClient(
+		apiKey,
+	)
+	if err != nil {
+		t.Fatal("Error creating client", err)
```

Comment:
@andrewoh531 probably better:`t.Fatalf("Could not create client: %v", err)`? `Could not` is a good convention for reporting errors.

Reply:
@andrewoh531 Actually, per incoming suggestions use lowercase, so `could not...`